ui/app/AppLayouts/Wallet/SendModal.qml

```diff
@@ -14,6 +14,12 @@ ModalPopup {
     title: qsTrId("command-button-send")
     height: 504
 
+    property var selectedRecipient
+    onSelectedRecipientChanged: {
+        selectRecipient.selectedRecipient = this.selectedRecipient
+        selectRecipient.readOnly = !!this.selectedRecipient && !!this.selectedRecipient.address
+    }
```

Comment:
What do you think about making the `selectedRecipient` property an alias of the `RecipientSelector` component? And any affected properties can be bound directly to its own properties. This is what I mean haha:
```suggestion
    property alias selectedRecipient: selectRecipient.selectedRecipient
    RecipientSelector {
        id: selectRecipient
        readOnly = !!selectedRecipient && !!selectedRecipient.address
    }
```